migrations/src/000003.ts

```diff
@@ -0,0 +1,59 @@
+import {getSelectResults} from "./src/utils";
+
+interface Attribute {
+  key: string;
+  value: string;
+}
+
+function selectEventsBatch (limit: number, offset: number) {
+  return `SELECT id, attributes FROM events LIMIT ${limit} OFFSET ${offset}`;
+}
+
+function insertEventAttributes(values) {
+  const columns = ["id", "key", "value", "event_id"];
+  return `INSERT INTO event_attributes (${columns.join(", ")}) VALUES ${values.join(",\n")}`;
+}
+
+export function migrationExpandEventAttributes() {
+  // @ts-ignore
+  plv8.execute("SET SCHEMA 'app'");
+
+  const batchSize = 2000;
+
+  // NB: batch operations to reduce memory footprint
+  for (let batch = 0;; batch++) {
+    const query = selectEventsBatch(batchSize, batchSize * batch);
+    // @ts-ignore
+    const events = getSelectResults(plv8.execute(query));
+
+    if (!events || events.length === 0) {
+      // NB: no more events
+      break;
+    }
+
+    // NB: process batch; `eventValues` order MUST match `columns`
+    const values = [];
+    for (const event of events) {
+      const [eventId, attributes] = event;
+      for (const [i, attr] of Object.entries(attributes as Attribute[])) {
+        const eventAttrValues = [];
+        // NB: must single quote values for proper SQL interpolation.
+        // attr.value already contains escaped quotes from
+        // JSON stringification in event handler.
+        eventAttrValues.push(`'${eventId}-${i}'`);
+        // TODO: check attr value "sanitization" func in event handler
```

Comment:
https://github.com/fetchai/ledger-subquery/blob/main/src/mappings/primitives.ts#L117 causes the value to be wrapped in double quotes, which subsequently get escaped and become part of the value in the DB. We can apply similar "un-quoting" logic as below, directly in the event handler to fix this quirk in the data going forward while still sanitizing non-text characters.